vpn-router-web/src/main/bundles/README.md

```diff
@@ -1,31 +1,26 @@
-This directory is automatically generated by Vaadin and contains the pre-compiled
+This directory is automatically generated by Vaadin and contains the pre-compiled 
 frontend files/resources for your project (frontend development bundle).
 
 It should be added to Version Control System and committed, so that other developers
 do not have to compile it again.
 
 Frontend development bundle is automatically updated when needed:
-
```

Comment:
You should not change the format of files

Reply:
Hm.... I didn't change this file manually, it was generated automatically

Reply:
You changed the file. You should not.

Reply:
https://github.com/KirillSmirnow/vpn-router/pull/36/files

Reply:
Now I've restored this file from the main branch. But I'm not sure it was a good idea)